Selective compilation of tedge-mapper crate fails

**Describe the bug**

Running `cargo check` from the repo root directory runs fine. But, building the mapper project separately with `cargo check -p tedge_mapper` , fails as follows:

```
Compiling tedge_mapper v0.6.0 (/home/rina/thinedge/thin-edge/crates/core/tedge_mapper)
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
  --> crates/core/tedge_mapper/src/main.rs:76:1
   |
76 | #[tokio::main]
   | ^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: could not find `process` in `tokio`
   --> crates/core/tedge_mapper/src/c8y/converter.rs:590:33
    |
590 |     let child = tokio::process::Command::new(&command)
    |                                 ^^^^^^^ not found in `tokio::process`
    |
help: consider importing one of these items
    |
1   | use clap::Command;
    |
1   | use std::process::Command;
    |

For more information about this error, try `rustc --explain E0433`.
error: could not compile `tedge_mapper` due to 2 previous errors
```

--- a/crates/core/tedge_mapper/Cargo.toml
+++ b/crates/core/tedge_mapper/Cargo.toml
@@ -49,7 +49,7 @@ tedge_utils = { path = "../../common/tedge_utils", features = ["logging"] }
 thin_edge_json = { path = "../thin_edge_json" }
 thiserror = "1.0"
 time = "0.3"
-tokio = { version = "1.8", features = ["rt", "sync", "time"] }
+tokio = { version = "1.8", features = ["process", "rt", "rt-multi-thread", "sync", "time"] }
 toml = "0.5"
 tracing = { version = "0.1", features = ["attributes", "log"] }
 

